### PR TITLE
Adjust style for action run list (align icons, adjust padding)

### PR DESCRIPTION
**Pull request by HesterG**
Main changes:

- Moved the icon into `action-item-main`, and make it `position: absolute` to allow add margin it to align with `issue-item-top-row`.
- Adjusted padding and color of texts.

# Before

Screen Shot 2023-06-09 at 17 04 38

# After

Screen Shot 2023-06-09 at 18 32 47

On hover:

Screen Shot 2023-06-09 at 18 32 54

### CODE REVIEW

**Comment by silverwind** [comment]
It does look better, thanks. I wonder of `position: absolute` can be avoided like we do on the issue/pr list.

File: web_src/css/shared/issuelist.css [hunk 1]
```diff
@@ -21,7 +21,20 @@
...
+}
+
+/* height of icon is 16px. and line height of issue-item-top-row is about 20px,
+  so need margin-top 2px to make them aligned 
+*/
+.action-item-icon {
+  position: absolute;
+  left: 0;
+  margin-top: 2px;
+}
```
Comment (silverwind) [inline web_src/css/shared/issuelist.css]: If you just need to re-align a element, `position: relative` and `top: 2px` has the same effect and is more compatible than `position: absolute` which can give unexpected results depending on where the closest relatively positioned parent is.

But before doing that, check how the issue list does it.

Comment (HesterG) [reply web_src/css/shared/issuelist.css]: I see, I reverted some unnecessary changes. The difference between issuelist and the run list is that the icon inside runlist is wrapped by `<span>`, so it has different `line-height` with `issue-item-top-row`, producing this `4px` offset.

Comment (silverwind) [reply web_src/css/shared/issuelist.css]: Issue icon has

```css
.issue.list > .item .issue-item-icon svg {
  margin-right: 0.75rem;
  margin-top: 1px;
}
```

I guess we can replicate/share this style.

Comment (HesterG) [reply web_src/css/shared/issuelist.css]: Removed styles, I found they could all share with issuelist, and updated screenshots

**Review by silverwind — APPROVED** [review]

File: web_src/css/shared/issuelist.css (hunk 1 above)
Comment (silverwind) [reply web_src/css/shared/issuelist.css]: Thanks, at some point we should probably check if we can even share the template if they are sufficiently similar, but does not need to be done here.

**Review by lunny — APPROVED** [review]

**Pull request merged**